<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>primera</class>
 <widget class="QMainWindow" name="primera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>980</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>147</green>
        <blue>174</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>147</green>
        <blue>174</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>147</green>
        <blue>174</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>147</green>
        <blue>174</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>147</green>
        <blue>174</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>147</green>
        <blue>174</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>147</green>
        <blue>174</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Centro de estetica</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/uco/POO/entornoVirtual/_92127008_thinkstockphotos-615094882.jpg</normaloff>../../Downloads/uco/POO/entornoVirtual/_92127008_thinkstockphotos-615094882.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="BotEntrar">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>660</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/uco/POO/entornoVirtual/_92127008_thinkstockphotos-615094882.jpg</normaloff>../../Downloads/uco/POO/entornoVirtual/_92127008_thinkstockphotos-615094882.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>951</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>60</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Centro de Estética</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultado">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>800</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="VarUsu">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>410</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="VarPass">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>520</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>510</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BorraPrograma">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>660</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cerrar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/uco/POO/entornoVirtual/_92127008_thinkstockphotos-615094882.jpg</normaloff>../../Downloads/uco/POO/entornoVirtual/_92127008_thinkstockphotos-615094882.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
